debian/changelog

```diff
@@ -1,3 +1,28 @@
+ubuntu-advantage-tools (29) mantic; urgency=medium
+
+  * d/README:
```

Comment:
I think all `d/` changes should be listed - including `control`, `postinst`, and `postrm` - even if the changes are simple enough or not user-related, we have been instructed in the past to make those explicit.